### PR TITLE
Bug 1500061 - Tighen CSP for new tab

### PR DESCRIPTION
**Pull request by k88hudson**
Note that the rest of snippets will be removed in a future patch.

### CODE REVIEW

**Comment by k88hudson** [comment]
@freddyb would you mind taking a look at this? So far I've only removed `unsafe-inline` for scripts

File: bin/render-activity-stream-html.js [hunk 1]
```diff
@@ -97,7 +97,7 @@ for (const src of ${JSON.stringify(scripts, null, 2)}) {
 <html lang="${options.locale}" dir="${options.direction}">
   <head>
     <meta charset="utf-8">
-    <meta http-equiv="Content-Security-Policy" content="default-src 'none'; script-src 'unsafe-inline' resource: chrome:; connect-src https:; img-src https: data: blob:; style-src 'unsafe-inline';">
+    <meta http-equiv="Content-Security-Policy" content="default-src 'none'; script-src resource: chrome:; connect-src https:; img-src https: data: blob:; style-src 'unsafe-inline';">
```
Comment (mozfreddyb) [inline bin/render-activity-stream-html.js]: Looks good!  :+1: 

The adjacent `connect-src` surprises me a bit. Do you actually use fetch, XHR, EventSource, sendBeacon,   `<a ping>` or Websockets?
To clarify, I found it just surprising. Not concerning :)

**Comment by Mardak** [comment]
Oh, and there was a comment from ckerschb about adding `object-src 'none'` https://bugzilla.mozilla.org/show_bug.cgi?id=1500061#c3

File: bin/render-activity-stream-html.js (hunk 1 above)
Comment (k88hudson) [reply bin/render-activity-stream-html.js]: Ah good catch – we used to use `fetch` from old snippets, but it's no longer needed. I supposed we can remove that

Comment (k88hudson) [reply bin/render-activity-stream-html.js]: Nevermind, still needed for one of the form submit actions in a template.

**Comment by Mardak** [comment]
Reverted this https://github.com/mozilla/activity-stream/commit/a3fe11f0f32415c98fa7a31b996735b4d580bfb1 for test failures when trying to export:
```
Console message: [JavaScript Error: "Content Security Policy: The page’s settings blocked the loading of a resource at inline (“script-src”)." {file: "about:newtab" line: 16}]
```
https://treeherder.mozilla.org/logviewer.html#/jobs?job_id=228060887&repo=try&lineNumber=1980-1987

Automation uses artifact builds. Try was failing on asan.

**Pull request merged**